distribution/docs/src/main/resources/_contents/_catalog-frameworks/json-definition-intro-contents.adoc

```diff
@@ -0,0 +1,33 @@
+=== JSON "Definition" Files.
```

Comment:
This section seems more related to metacards and metacard attributes more than the catalog framework. Do we have a more appropriate place for this section?